scripts/format.js

```diff
@@ -1,25 +1,3 @@
-const path = require('path');
-const chalk = require('chalk');
-
 const prettierWrite = require('../lib/runPrettier').write;
-const prettierConfig = require('../config/prettier/prettierConfig');
-
-const builds = require('../config/builds');
-
-const args = require('../config/args').argv;
-
-console.log(chalk.cyan('Formatting source code with Prettier'));
-
-const filePattern =
-  args.length === 0
-    ? builds[0].paths.src.map(srcPath => `${srcPath}/**/*.js`)
-    : args;
 
-prettierWrite(filePattern, prettierConfig)
-  .then(() => {
-    console.log(chalk.cyan('Prettier format complete'));
-  })
-  .catch(exitCode => {
-    console.error('Error: Prettier exited with exit code', exitCode);
-    process.exit(1);
-  });
+(async () => await prettierWrite())();
```

Comment:
Is this any different to just calling `prettierWrite()`? Node will wait for the event loop to be empty before exiting, and you'll get an `UnhandledPromiseRejection` error if anything throws.

Not a big deal, just curious!

Reply:
No i believe you are correct, that's essentially a relic of refactoring as the file became smaller and smaller 👏